src/Apps/MyCollectionInsights/Routes/MyCollectionInsightsMedianSalePriceAtAuction/MyCollectionInsightsMedianSalePriceAtAuction.tsx

```diff
@@ -9,6 +23,44 @@ export const MyCollectionInsightsMedianSalePriceAtAuction = () => {
       <Text variant={["xs", "sm-display"]} color="black60">
         Track price stability or growth of your artists
       </Text>
+
+      <Spacer mb={[2, 4]} />
+
+      <Flex
+        flexDirection={"row"}
+        justifyContent={["space-between", "flex-start"]}
+      >
+        <EntityHeaderArtistFragmentContainer
+          artist={artistData}
+          FollowButton={<Fragment></Fragment>}
+          displayLink={false}
+        />
+
+        <Spacer ml={4} />
+
+        <Clickable
+          color="black60"
+          textDecoration="underline"
+          onClick={() => setShowSelectArtistModal(!showSelectArtistModal)}
+        >
+          Change Artist
+        </Clickable>
+      </Flex>
+
+      {showSelectArtistModal && (
+        <ModalDialog
+          onClose={() => setShowSelectArtistModal(false)}
+          width={"100%"}
```

Comment:
**Suggestion:** Maybe we should limit the width on desktop view? let's say 800px? and 100% on mWeb?

```suggestion
          width={["100%", 800]}
```

Reply:
good idea, I will do it and follow up later with Dascha